jina/types/ndarray/dense/numpy.py

```diff
@@ -8,6 +8,8 @@
 
 __all__ = ['BaseDenseNdArray']
 
+QUANTIZE = os.environ.get('JINA_ARRAY_QUANT')
```

Comment:
I think is better to have this as class attribute so that remote jinas can reset this and just be computed at the first import